interim-jan-2021/slides-directories.md

```diff
@@ -0,0 +1,30 @@
+# Scenario 3: Directories of Encrypted Resolvers
+
+Three Parties:
+* Publisher: Curates a list of distinct resolvers
+* Client: Fetches the list from a trusted source
+* Resolvers: Identified in the list, ready for access by the client
+------------------------------------------------
+# Example use cases
+
+* A browser vendor wants to provide users with a list of resolvers to consider, curated by a trusted third party
+* An OS vendor wants to keep its list of trusted resolvers current without requiring a software update
+* A user wants to choose a resolver from a list offered by a network operator who they trust
+------------------------------------------------
+# Requirements
+
+* A list can be published by a trusted network
+* A list can be published at an HTTP URL
+* Each resolver controls its own self-description
+* Suitable for use in an onscreen interactive menu
+* Can be used as an additional safeguard for untrusted upgrade instructions
+* Uses the same protocols as the previous scenarios
```

Comment:
From @jim-add

```suggestion
* Uses the same protocols as the previous scenarios
	* Fail safe fallback(s) and least surprise if/when a suitable resolver isn't on the list
* Verification of list info (certs? DNSSEC?)
```

Reply:
@jim-add

For fail-safe fallback, I'm not sure what use case you're considering, but it sounds like it requires a lot more than just a "list of resolvers", which is all that's being proposed here.

For verification, I'm not sure what you have in mind.  Do you mean "the directory provides an identifier suitable for authentication of the encrypted resolver"?  I agree with that, although I think it's implied.